scripts/tests_markdown.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env python
+import argparse
+import json
+import os
+import statistics
+import sys
+
+import jinja2
+from mir import html_report
+
+
+DESCRIPTION = "Script to generate (potentially merged) HTML summary of Galaxy Test Performance"
+templateLoader = jinja2.FileSystemLoader(searchpath="./scripts")
+template_env = jinja2.Environment(loader=templateLoader)
+TEMPLATE_FILE = "tests_markdown.tpl"
+TEMPLATE_COMPARE_FILE = "tests_markdown_compare.tpl"
+LINKS = [
+    {"href": "https://github.com/galaxyproject/galaxy", "title": "Galaxy"}
+]
+
+
+def main(argv=None):
+    if argv is None:
+        argv = sys.argv[1:]
+
+    args = _parser().parse_args(argv)
+    common_env = {
+        "include_raw_metrics": args.include_raw_metrics,
+    }
+    if len(args.input_path) == 1:
+        markdown_output = _summarize_one(args.input_path[0], common_env)
+    else:
+        markdown_output = _summarize_many(args.input_path, common_env)
+
+    html_output = html_report(markdown_output, args.title, links=LINKS)
+    with open(args.output_path, "w") as f:
+        f.write(html_output)
+
+
+def _summarize_one(input_path, common_env):
+    environment = _prepare_raw_data(input_path)
+    environment.update(common_env)
+    template = template_env.get_template(TEMPLATE_FILE)
+    output = template.render(environment)
+    return output
+
+
+def _summarize_many(input_paths, common_env):
+    raw_data_dicts = []
+    for arg in input_paths:
+        raw_data_dicts.append(_prepare_raw_data(arg))
+    environment = _merge_summarizes(raw_data_dicts)
+    environment.update(common_env)
+    template = template_env.get_template(TEMPLATE_COMPARE_FILE)
+    output = template.render(environment)
+    return output
+
+
+def _empty_statistics():
+    return {"count": 0, "sum": "n/a", "median": "n/a", "stdev": "n/a", "mean": "n/a"}
```

Comment:
```suggestion
    return {"count": 0, "sum": 0, "median": None, "stdev": None, "mean": None}
```
This is relevant *only* if you make the formatting change in the template (see suggestion below), this needs to be reset to the proper values. I think 0 is appropriate for `sum`, but maybe `None` makes more sense?

Reply:
I kind of wish I wouldn't have - there is not a clean way to do the map and it seems like they prefer formatting before you hit jinja2. https://github.com/pallets/jinja/pull/906#issuecomment-548393215.

I'm going to try to rework this to do the formatting for the fix precision but not do the formatting in the template.

Reply:
Oh, I see. Sorry for suggesting that route! And yes, it's not like we use that data in the template in different ways, so it might as well come formatted as a string.